services/records/record_templates/parsers.go

```diff
@@ -0,0 +1,109 @@
+package record_templates
+
+import (
+	"github.com/leanspace/terraform-provider-leanspace/helper"
+	"github.com/leanspace/terraform-provider-leanspace/helper/general_objects"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func (recordTemplate *RecordTemplate) ToMap() map[string]any {
+	resourceMap := make(map[string]any)
+	resourceMap["id"] = recordTemplate.ID
+	resourceMap["name"] = recordTemplate.Name
+	resourceMap["description"] = recordTemplate.Description
+	resourceMap["record_state"] = recordTemplate.RecordState
```

Comment:
The api has a state field doesn't it?
```suggestion
	resourceMap["state"] = recordTemplate.RecordState
```